app/frontend/src/components/FeedView.tsx

```
@@ -135,25 +163,39 @@ export default function FeedView({ highlightedPostId, className = '' }: FeedView
     }
-  });
+
+    const filtered = feed.filter(post => {
+      switch (activeFilter) {
+        case 'following':
+          // In a real app, this would check if the user follows the post author
+          return true; // For now, show all posts
+        case 'trending':
+          // In a real app, this would filter by trending algorithm
+          // For now, use a simple heuristic based on post age and tags
+          const postAge = Date.now() - new Date(post.createdAt).getTime();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)